### PR TITLE
[search] clusterize-postcodes converts only english letters to A.

### PR DESCRIPTION
**Pull request by ygorshenin**
@mgsergio or @mpimenov PTAL

### CODE REVIEW

File: search/search_quality/clusterize-postcodes.lisp [hunk 1]
```diff
@@ -27,16 +27,20 @@ exec /usr/bin/env sbcl --noinform --quit --load "$0" --end-toplevel-options "$@"
...
   (string-trim *seps* string))
 
+(defun english-letter-p (char)
+  (or (and (char>= char #\a) (char<= char #\z))
+      (and (char>= char #\A) (char<= char #\Z))))
+
 (defun get-postcode-pattern (postcode)
   "Simplifies postcode in the following way:
-   * all letters are replaced by 'A'
+   * all english letters are replaced by 'A'
```
Comment (mpimenov) [inline search/search_quality/clusterize-postcodes.lisp]: latin?

Comment (ygorshenin) [inline search/search_quality/clusterize-postcodes.lisp]: Done

**Comment by ygorshenin** [comment]
@mpimenov PTAL

**Comment by mpimenov** [comment]
LGTM

**Pull request merged**